tests/situations/commun/modeles/evenement.js

```diff
@@ -1,8 +1,8 @@
 import Evenement from 'commun/modeles/evenement';
 
 describe("l'événement", function () {
-  it("renvoit une exception lorsqu'on lui demande son nom", function () {
-    expect(new Evenement().nom).to.throwError('Pas implémenté');
+  it('renvoit son nom', function () {
```

Comment:
je chipotte mais tant qu'à réécrire le `it` je propose de corriger la faute : `il renvoie`